app/controllers/pulitzer/posts_controller.rb

```diff
@@ -0,0 +1,46 @@
+class Pulitzer::PostsController < Pulitzer::ApplicationController
+  before_filter :get_post, only: [:show, :edit, :update]
+
+  def index
+    @posts = Pulitzer::Post.where(post_type_id: params[:post_type_id])
+  end
+
+  def new
+    @post = Pulitzer::Post.new(post_type_id: params[:post_type_id])
+    render partial: 'new', locals: { post: @post }
+  end
+
+  def create
+    @post = Pulitzer::Post.create(post_params)
+    render partial: 'show_wrapper', locals: { post: @post }
+  end
+
+  def show
+    render partial: 'show', locals: { post: @post }
+  end
+
+  def edit
+    render partial: 'form', locals: { post: @post }
+  end
+
+  def update
+    @post.update_attributes(post_params)
+    render partial: 'show', locals: { post: @post }
+  end
+
+  def destroy
+    @post.destroy
+    render nothing: true
```

Comment:
Food for thought: if this is a normal (non-ajax) request I think we probably want to redirect to some index? and if it's ajax shouldn't we respond with head :ok instead?